cnames_active.js

```diff
@@ -1458,8 +1458,9 @@ var cnames_active = {
   "ksp": "keepsobp.github.io",
   "kst": "lucaelin.github.io/KST", // noCF
   "ktm": "developers-nepal.github.io/ktmjs",
+  "kubernetes":"theinternetfolks.github.io/kubernetes",
   "kucos": "kucosjs.github.io",
-  "kumeru": "ultirequiem.github.io/kumeru"
+  "kumeru": "ultirequiem.github.io/kumeru",
```

Comment:
```suggestion
  "kumeru": "ultirequiem.github.io/kumeru"
```
Please revert this so that we have a consistent and clean git blame :)